### ISSUE TITLE
How can I reproduce EMANet's results on pascal voc data set?

### ISSUE THREAD

**Issue by Nastu-Ho** [open-mmlab/mmsegmentation#3009]
I found no config for voc in the emanet directory. So I consulted deeplabplus's config on voc.
![image](https://github.com/open-mmlab/mmsegmentation/assets/39642013/cd357a34-8f25-4f22-9676-c4c437124fe1)

Resnet-101 was chosen as the backbone.
Everything else follows the default Settings.
I trained with 4 V100 GPU in parallel, sample-per-gpu set to 4.
During the training process, the performance of the validation set can only achieve 72.x mIoU at most, which is very different from the 80.x mIoU in the paper.
